extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/controlBar.js

```diff
@@ -112,13 +113,28 @@ var ControlBar = React.createClass({
     } else {
       this.toggleQualityPopover();
       this.closeCaptionPopover();
+      this.closeConfigPopover();
     }
   },
 
-  toggleQualityPopover: function() {
```

Comment:
I think popover is one word
https://v4-alpha.getbootstrap.com/components/popovers/